packages/app/components/Today/ChatbotToday.tsx

```diff
@@ -81,6 +81,9 @@ export const ChatbotToday: React.FC = () => {
           ])
         })
       })
+      .catch((err) => {
+        console.error(err)
```

Comment:
probably better to `throw err`?

Reply:
If we don't catch the error, it causes that annoying uncaught error exception whenever we go on the Today page and we don't have chatbot running. 
Plus this is only for the suggested questions, which i don't think is a critical component that's worth informing us if it throws a 500 error.